test_pull_requests

```diff
@@ -458,6 +460,22 @@ module Hub
       comment += extended_tests if extended_tests
       recreate_comment(pull_id, bot_comment['id'], repo, comment)
 
+      bot_sha_comment = get_comment_with_prefix(pull_id, repo, BASE_SHA_PREFIX)
+      base_sha_command = %{
```

Comment:
Again, you are going to have to be able to do this with an api call for the merge side.  I also wouldn't use git config.  Just write the value to the bot comment.  That's where it needs to be for merge anyway.